detections/network/detect_dns_data_exfiltration_using_pretrained_model_in_dsdl.yml

```diff
@@ -0,0 +1,95 @@
+name: Detect DNS Data Exfiltration using pretrained model in DSDL
+id: 92f65c3a-968c-11ed-a1eb-0242ac120012
+version: 1
+date: '2023-04-27'
+status: experimental
+author: Abhinav Mishra, Kumar Sharad and Namratha Sreekanta, Splunk
+type: Anomaly
+data_source: []
+description: The following analytic uses a pre trained deep learning model to
+  detect DNS data exfiltration. The model is trained on the data we collected
+  and is inferred on live data. This detection detects low throughput DNS
+  Tunneling (data exfiltration) using features computed from past events between
+  the same src and domain. The search uses macros from URL ToolBox app to
+  generate features used by the model. The model is a deep learning model that
+  accepts DNS request as input along with a few custom features to generate a
+  pred_is_exfiltration_proba score. The higher the pred_is_exfiltration_proba,
+  the more likely the DNS request is data exfiltration. The threshold for
+  flagging a request as DNS exfiltration is set at 0.5.
+search: '| tstats `security_content_summariesonly` count from
+  datamodel=Network_Resolution by DNS.src _time DNS.query |
+  `drop_dm_object_name("DNS")` | sort - _time,src, query | streamstats count as
+  count by src query | where count < 10 | table src,query,count,_time | apply
+  detect_dns_data_exfiltration_using_pretrained_model_in_dsdl_filter | table
+  src,_time,
+  query,count,pred_is_dns_data_exfiltration_proba,pred_is_dns_data_exfiltration
+  | where count == 1 | rename pred_is_dns_data_exfiltration_proba as
+  is_exfiltration_score | rename pred_is_dns_data_exfiltration as
+  is_exfiltration | where is_exfiltration_score > 0.5 |
+  `security_content_ctime(_time)`| rename src as "Source IP" _time as "time" |
```

Comment:
i would remove this rename :  `rename src as "Source IP" _time as "time"` and instead output the original field names

Reply:
I have the same thing for detecting suspicious txt records - will make change to that as well